Source/WarpX.cpp

```diff
@@ -768,7 +768,7 @@ WarpX::ReadParameters ()
             use_kspace_filter = use_filter;
             use_filter = false;
         }
-        else // FDTD
+        else if (WarpX::electromagnetic_solver_id != ElectromagneticSolverAlgo::HybridPIC)
```

Comment:
Could you please briefly explain why r-filtering in RZ should be allowed for the hybrid-solver but not the Maxwell FDTD solver?

Reply:
I was using it to help with source smoothing. It currently does not use radial weights in R, so there is some non-conservation of J and rho in r when using a straight bilinear filter radially on the density fields. I am planning to try to fix this to add the geometric factor, but this will be a follow-up PR. I can add a warning stating this and to use at your own risk, but I did not want to change the behavior of the FDTD solver until I can work with Remi on it. Issue here: https://github.com/ECP-WarpX/WarpX/issues/1943